hail_search/hail_search_query.py

```diff
@@ -49,11 +52,20 @@ class BaseHailTableQuery(object):
 
     GENOME_VERSIONS = [GENOME_VERSION_GRCh38]
     GLOBALS = ['enums']
+    TRANSCRIPTS_FIELD = None
     CORE_FIELDS = [XPOS]
     BASE_ANNOTATION_FIELDS = {
         'familyGuids': lambda r: r.family_entries.filter(hl.is_defined).map(lambda entries: entries.first().familyGuid),
+        'genotypeFilters': lambda r: hl.str(' ,').join(r.filters),
+    }
+    ENUM_ANNOTATION_FIELDS = {
+        'transcripts': {
+            'response_key': 'transcripts',
+            'empty_array': True,
+            'format_value': lambda value: value.rename({k: _to_camel_case(k) for k in value.keys()}),
+            'format_values': lambda values: values.group_by(lambda t: t.geneId),
```

Comment:
Might want to name this function something other than `format_values`.  Having `format_value` and `format_values` both be present but have different behavior is a bit surprising.